.packit.yml

```diff
@@ -21,7 +21,7 @@ actions:
     # Drop backported patches from the specfile, but keep the downstream-only ones
     # - Patch0000-0499: backported patches from upstream
     # - Patch0500-9999: downstream-only patches
-    - "sed -ri '/^Patch0[0-4][0-9]{2}+\\:.+\\.patch/d' .packit_rpm/systemd.spec"
+    - "sed -ri '/^Patch0[0-4]?[0-9]{0,2}\\:.+\\.patch/d' .packit_rpm/systemd.spec"
```

Comment:
The assumption here was that we'd use appropriate prefixes. Maybe let's just instead add immediately fails patch names do not conform.

Reply:
I though to, since the "offending" patch wasn't added by you. Anyway, since the patch is still in a *valid range*, I'd accept it, even though it doesn't conform to the expected name format.

Reply:
This seems to work, so let's merge.